daemon-tests/tests/main/order.rs

```diff
@@ -189,6 +198,9 @@ async fn taker_places_order_for_same_offer_twice_results_in_two_cfds() {
     );
 }
 
+// TODO: Potentially create a test where we take one ethusd and one btc usd order to ensure both can
+// be set up and we receive both on the cfd feed
+
 /// To be used for the first contract setup
 async fn first_contract_setup(maker: &mut Maker, taker: &mut Taker, order_id: OrderId) {
```

Comment:
🔧 What is a "first" contract setup? Every contract setup is the first and only one. Can we rename this?

Reply:
for some reason Daniel has created 2 functions - one for the "first" and then for "additional" contract setup. I'll investigate whether we need both and remove one if not needed.

I'm surprised by this too.

Reply:
it turns out that one was enough. incorporated it in a commit in this PR.